### PR TITLE
chore: add GitHub Codespace containers

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Other information

Adds a devcontainer with some default configuration.
I also tried to add a specific container for `ngrx.io` but I don't see it anywhere, maybe if it's pushed to master that it becomes visible 🙃

### CODE REVIEW

**Review by markostanimirovic — APPROVED** [review]
🎉

File: .devcontainer/ngrx.io/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/typescript-node
+{
+  "name": "ngrx.io",
+  "workspaceMount": "type=bind,src=${localWorkspaceFolder},dst=/projects/ngrx.io,consistency=delegated",
```
Comment (timdeschryver) [inline .devcontainer/ngrx.io/devcontainer.json:5]: I found the screen, but it appears to not be working.
Putting this PR on draft.

image

**Comment by timdeschryver** [comment]
It seems like opening a specific folder isn't supported yet (or I'm doing something wrong 😅)
For now, we still have a separate devcontainer for it because it contains different plugins, and a `yarn install` is also executed for the ngrx.io project.

File: .devcontainer/ngrx.io/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/typescript-node
+{
+  "name": "Platform",
```
Comment (timdeschryver) [inline .devcontainer/ngrx.io/devcontainer.json:4]:
```suggestion
  "name": "Docs (ngrx.io)",
```

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**